### ISSUE TITLE
Icons used in CSS pseudo element is read by screen readers

### ISSUE THREAD

**Issue by its-me-mario** [az-digital/arizona-bootstrap#1777]
## Problem/Motivation
Decrotive icons used in CSS pseudo-content is read by screen readers.

### Describe the bug
When icons are used in CSS pseudo-content, screen readers will read whatever is set in the content. Since the icons are often decrotive, this might cause confusion.
This issue was first mentioned (Oct 22, 2025) by Jeff at DRC when reviewing a NetID page.

### To Reproduce
Using Jaws on any platform/browser (also on Mac Voice Over) read the breadcrumb content for a page that is at least two levels deep and is showing a chevron in the breadcrumb.

## Proposed resolution
Some possible options:
- use an inline image E.g. <svg> or <img>
- use an encoded svg as the CSS pseudo-content with currentColor to inherit the font color
- use clip-path in the CSS with: currentColor to inherit the font color
- use an SVG for the background (not ideal)
[I have some examples from long ago](https://codepen.io/itsmemario/pen/WNObzwq)

### Expected behavior
It should be optional to have the icon read, or not be read at all because they often don't provide value.

##  Screenshots
[screenshot](https://limewire.com/d/v3ls5#Hm6UnkmtZd)
